### ISSUE TITLE
Adding interpolation to a GoogleTiles add_image call cause savefig call to fail at the end of file

### ISSUE THREAD

**Issue by andrewkovachik** [SciTools/cartopy#1579]
### Description
I tried following this stack post to get better resolution of google image tiles

https://stackoverflow.com/questions/49155110/why-do-my-google-tiles-look-poor-in-a-cartopy-map

Python version
```
python2 -V
Python 2.7.17
```

#### Code to reproduce

```

import matplotlib.pyplot as plt
from matplotlib.transforms import offset_copy
 
import cartopy.crs as ccrs
import cartopy.io.img_tiles as cimgt


def main():
    # Create a Stamen terrain background instance.

    stamen_terrain = cimgt.GoogleTiles()

    fig = plt.figure()

    # Create a GeoAxes in the tile's projection.
    ax = fig.add_subplot(1, 1, 1, projection=stamen_terrain.crs)
     ## Limit the extent of the map to a small longitude/latitude range.
    ax.set_extent([-22, -15, 63, 65])
     ## Add the Stamen data at zoom level 8.
    ax.add_image(stamen_terrain, 8, interpolation='bilinear')
    plt.savefig("test.png")
 if __name__ == '__main__':
    main()

```

#### Traceback 

```
Traceback (most recent call last):
  File "gmexample.py", line 28, in <module>
    main()
  File "gmexample.py", line 25, in main
    plt.savefig("test.png")
  File "/home/andrew/.local/lib/python2.7/site-packages/matplotlib/pyplot.py", line 695, in savefig
    res = fig.savefig(*args, **kwargs)
  File "/home/andrew/.local/lib/python2.7/site-packages/matplotlib/figure.py", line 2062, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/home/andrew/.local/lib/python2.7/site-packages/matplotlib/backend_bases.py", line 2263, in print_figure
    **kwargs)
  File "/home/andrew/.local/lib/python2.7/site-packages/matplotlib/backends/backend_agg.py", line 517, in print_png
    FigureCanvasAgg.draw(self)
  File "/home/andrew/.local/lib/python2.7/site-packages/matplotlib/backends/backend_agg.py", line 437, in draw
    self.figure.draw(self.renderer)
  File "/home/andrew/.local/lib/python2.7/site-packages/matplotlib/artist.py", line 55, in draw_wrapper
    return draw(artist, renderer, *args, **kwargs)
  File "/home/andrew/.local/lib/python2.7/site-packages/matplotlib/figure.py", line 1493, in draw
    renderer, self, artists, self.suppressComposite)
  File "/home/andrew/.local/lib/python2.7/site-packages/matplotlib/image.py", line 141, in _draw_list_compositing_images
    a.draw(renderer)
  File "/home/andrew/.local/lib/python2.7/site-packages/matplotlib/artist.py", line 55, in draw_wrapper
    return draw(artist, renderer, *args, **kwargs)
  File "/home/andrew/.local/lib/python2.7/site-packages/cartopy/mpl/geoaxes.py", line 479, in draw
    return matplotlib.axes.Axes.draw(self, renderer=renderer, **kwargs)
  File "/home/andrew/.local/lib/python2.7/site-packages/matplotlib/artist.py", line 55, in draw_wrapper
    return draw(artist, renderer, *args, **kwargs)
TypeError: draw() got an unexpected keyword argument 'interpolation'

```

<details>
 <summary>Full environment definition</summary>

### Operating system
`Linux Galileo 4.15.0-20-generic #21-Ubuntu SMP Tue Apr 24 06:16:15 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux`

### Cartopy version
>>> cartopy.__version__
u'0.18.0'

### pip list

```
enum (2.2.3)
arrow (0.15.6)
asn1crypto (0.24.0)
attrs (17.4.0)
Automat (0.6.0)
backports.functools-lru-cache (1.6.1)
basemap (1.2.1)
Cartopy (0.18.0)
certifi (2018.11.29)
cffi (1.11.5)
cftime (1.1.3)
chardet (3.0.4)
click (7.1.2)
configobj (5.0.6)
configparser (4.0.2)
constantly (15.1.0)
cryptography (2.5)
cycler (0.10.0)
Cython (0.29.19)
deluge (1.3.15)
enum34 (1.1.6)
Flask (1.1.2)
funcsigs (1.0.2)
futures (3.3.0)
geos (0.2.2)
glob2 (0.7)
hyperlink (17.3.1)
idna (2.8)
incremental (16.10.1)
ipaddress (1.0.22)
itsdangerous (1.1.0)
Jinja2 (2.11.2)
keyboard (0.13.2)
kiwisolver (1.1.0)
lxml (4.5.1)
MarkupSafe (1.1.1)
matplotlib (2.2.5)
mock (2.0.0)
netCDF4 (1.5.3)
nose (1.3.7)
numpy (1.16.6)
pandas (0.23.4)
paramiko (2.0.0)
pbr (3.1.1)
pexpect (4.2.1)
Pillow (6.2.2)
pip (9.0.1)
proj (0.2.0)
pyasn1 (0.4.2)
pyasn1-modules (0.2.1)
pycairo (1.16.2)
pycparser (2.19)
pygobject (3.26.1)
PyJWT (1.7.1)
pyOpenSSL (19.0.0)
pyparsing (2.4.7)
pyparted (3.11.1)
pyproj (2.2.2)
pyshp (2.1.0)
python-apt (1.6.4)
python-dateutil (2.8.1)
python-libtorrent (1.1.5)
python-xlib (0.20)
pytz (2020.1)
pyxdg (0.25)
requests (2.21.0)
scipy (1.2.3)
service-identity (16.0.0)
setproctitle (1.1.10)
setuptools (44.1.1)
Shapely (1.7.0)
six (1.15.0)
subprocess32 (3.5.4)
twilio (6.23.1)
Twisted (17.9.0)
urllib3 (1.24.1)
Werkzeug (1.0.1)
wheel (0.34.2)
yapf (0.25.0)
zope.interface (4.3.2)

```

</details>

**Comment by ajdawson** [comment]
I'm closing this as a duplicate of #1563 and #1567. An 0.18.1 release needs to be made before the fix is available for most users.

**Comment by andrewkovachik** [comment]
Hi, sorry about this. Is there a branch available with the fix? I ask only cause my memory is bad and I'll probably forget about this if I don't look into it now. As well, it may be worth leaving a comment on the mentioned stack post as right now its the first and only result on the search of the error and it is currently inaccurate.

Thanks a tone for working on this package, by the way, I love it.

**Comment by ajdawson** [comment]
The fix is merged in the v0.18.x branch
